### PR TITLE
Added new freecodecamp links

### PR DESCRIPTION
**Pull request by PiousSutherland**
AWS one I removed was 3 years old and needed updating. Added new neural net course as well.

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -1290,6 +1291,7 @@
 * [Stanford University Machine Learning](https://www.coursera.org/learn/machine-learning)
 * [Statistics for Applications](https://ocw.mit.edu/courses/18-650-statistics-for-applications-fall-2016/) - MIT OpenCourseWare
 * [Understand Machine Learning Engineering by Building Projects](https://github.com/alexeygrigorev/mlbookcamp-code/tree/master/course-zoomcamp) - Alexey Grigorev
+* [Understanding AI from Scratch – Neural Networks Course](https://www.youtube.com/watch?v=VgzHT9quo5c) - Dr Radu (freecodecamp)
```
Comment (eshellman) [inline courses/free-courses-en.md:1294]:
```suggestion
* [Understanding AI from Scratch – Neural Networks Course](https://www.youtube.com/watch?v=VgzHT9quo5c) - Radu (FreeCodeCamp)
```
We do not include honorifics.

**Review by eshellman — CHANGES_REQUESTED** [review]

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -413,7 +413,8 @@
 
 #### AWS
 
-* [AWS Certified Cloud Practitioner Course](https://www.youtube.com/playlist?list=PLt1SIbA8guuvfvUDVLpJepmbnYpOfYCIB) - Stephane Maarek
+* [AWS Certified Cloud Practitioner Course (CLF-C02)](https://www.youtube.com/watch?v=NhDYbskXRgc&t=7s&pp=ygUQZnJlZWNvZGVjYW1wIGF3cw%3D%3D) - Andrew Brown (freecodecamp)
```
[the remark this answers is not in the document]
Comment (PiousSutherland) [reply courses/free-courses-en.md:416]: This is 14 hours...
It _definitely_ counts as a course.

Honest mistake.

File: courses/free-courses-en.md [hunk 3]
```diff
@@ -413,7 +413,8 @@
 
 #### AWS
 
-* [AWS Certified Cloud Practitioner Course](https://www.youtube.com/playlist?list=PLt1SIbA8guuvfvUDVLpJepmbnYpOfYCIB) - Stephane Maarek
```
[the remark this answers is not in the document]
Comment (PiousSutherland) [reply courses/free-courses-en.md:416]: Is its being outdated not a valid argument?

[the remark this answers is not in the document]
Comment (eshellman) [reply courses/free-courses-en.md:416]: You could provide example of why the older one is no longer useful. Or you may also add a year to help disambiguate:
[AWS Certified Cloud Practitioner Course (YYYY)]

File: courses/free-courses-en.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (eshellman) [reply courses/free-courses-en.md:416]: you didn't view it, did you?

[the remark this answers is not in the document]
Comment (eshellman) [reply courses/free-courses-en.md]: Sorry, I think the lead-in ad fooled me on the timing.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Comment by PiousSutherland** [comment]
Thanks! I hope to contribute more soon, within the rules, of course.

**Pull request merged**